### DISCUSSION TITLE
在 自定义目录树下 支持 自定义框选单元格

### DISCUSSION THREAD

**Discussion by Vaner42** [antvis/S2#2135] [Show and tell]
我发现 s2 原本的自定义目录树或树形模式，在收起父级的时候框选父级，展开父级后发现 其子级并没有被选中
![1679732734016](https://user-images.githubusercontent.com/94883874/227985994-a6d7d282-6344-4509-84f3-32295589fd49.png)
![1679732869430](https://user-images.githubusercontent.com/94883874/227986116-2e60bccd-c0b8-423e-a712-4752f720f868.png)

所以 根据 s2 原有的方法增加了 自定义框选单元格，以及根据框选范围 复制、粘贴、删除 的方法。
demo体验地址：https://idd5pr.csb.app/
demo源码：https://codesandbox.io/s/tender-matan-ev6czl?file=/index.js
具体的改造思路 我写在掘金了，地址是 https://juejin.cn/post/7215200495984312357

**Comment by lcx-seima** [comment]
👏🏻

**Reply by infinityboy** [reply]
这个功能2.0版本包括了么？
